Anature/src/application/anatures/Anature.java

```diff
@@ -188,7 +166,7 @@ void setAbility(IAbility iAbility)
 			throw new IllegalArgumentException("Passed value \"ability\" was null.");
 		}
 
-		// TODO Should we add the ability to adding the NullAbility to the Anature
+		// TODO Should we add the ability to add the NullAbility to the Anature?
```

Comment:
No I don't think so. I figure that a Null ability should never be assigned and should only be used as a backup incase something goes wrong at creation time.